### ISSUE TITLE
Links in `Contribute Page` Not Working

### ISSUE THREAD

**Issue by Tanmoy741127** [CircuitVerse/mobile-app#260]
**Describe the bug**
Every links in `Contribute Page` is not working.
It includes 
- `Email Us` Button
- `Join Slack` Button 
- `Contribute to Open Source Github` Button
- `Become a Patron` Button
- `Donate Through Paypal` Button

**To Reproduce**
1. Open the app
2. Open Sidebar
3. Click on `Contribute` Button
4. Scroll & Click on the buttons to check the issues

**Expected behavior**
1. On click `Email Us` button, it should open  default mail application of device with mail id **support@circuitverse.org**
2. On click `Join Us Slack` button, it should open the invitataion link in browser 
3. On click `Contribute to Open Source Github` button, open the https://github.com/CircuitVerse/CircuitVerse link in browser
4. On click `Become a Patron`button, open https://www.patreon.com/CircuitVerse link in browser
5. On click `Donate Through Paypal` button, open https://www.paypal.me/satviksr link in browser

**Screenshots**
| upper | lower |
| ----------------- |--------------- |
| <img src="https://user-images.githubusercontent.com/57363826/221501085-a5d6cbd5-129d-4e20-ba6e-a86423f10f50.png" width="250px" ></img> | <img src="https://user-images.githubusercontent.com/57363826/221500866-8c8bf721-f5e0-4253-8311-e4b36f19c337.png" width="250px" ></img> |

**Device Information [optional]:**

- OS: Android

**Are you working on this issue? (Yes/No)**
No

**Comment by Aditya-150** [comment]
I want to work on this, can I take this?

**Comment by Tanmoy741127** [comment]
> I want to work on this, can I take this?

This issues is going to be part of FossHack 2023, You can register here https://fossunited.org/fosshack/2023/partner-projects-list/CircuitVerse

**Comment by Aditya-150** [comment]
Thanks for the info.

**Comment by Aditya-150** [comment]
> > Thanks for the info.
> 
> You can work on this!


Thanks🚀

**Comment by harshsharma-11** [comment]
@Aditya-150 bro any feedback does you able to fix it othervise i want to work on it 
@vedant-jain03

**Comment by Aditya-150** [comment]
You can work on it. I couldn't make it.

**Comment by harshsharma-11** [comment]
@Aditya-150 thanks mate happy coding.........

**Comment by harshsharma-11** [comment]
@Aditya-150 bro does you find any of above bug

**Comment by harshsharma-11** [comment]
@tanmoysrt except github of circuitverse all are working fine ,should i fix github link as instead of taking to circuitvrese repository it take to circuitverse profile

**Comment by tanmoysrt** [comment]
> @tanmoysrt except github of circuitverse all are working fine ,should i fix github link as instead of taking to circuitvrese repository it take to circuitverse profile

I think, if the link is working, it's better to keep it same.
If you like to change, I think other maintainers can provide better clarification

**Comment by harshsharma-11** [comment]
@vedant-jain03 is this valid if yes please assign it to me ,otherwise close this issue if not valid

**Comment by tanmoysrt** [comment]
> @vedant-jain03 the email and slack icons in mobile is not working at alll..............

Yup you can attach the same url that has in the circuitverse.org about section

**Comment by harshsharma-11** [comment]
> > @vedant-jain03 the email and slack icons in mobile is not working at alll..............
> 
> Yup you can attach the same url that has in the circuitverse.org about section

okk

**Comment by RAJEEVTIWARI8052** [comment]
plz assign this issue to me , i want to work on it

/assign

**Comment by JatsuAkaYashvant** [comment]
Closing this issue, already implemented for the contributor page.
